graviti/portex/factory.py

```diff
@@ -392,7 +400,10 @@ def __call__(self, **kwargs: Any) -> List[Any]:
             The applied list.
 
         """
-        return list(factory(**kwargs) for factory in self._factories)
+        array = list(factory(**kwargs) for factory in self._factories)
+        for factory in self._unpack_factories:
+            array.extend(factory(**kwargs))
```

Comment:
which means the unpacked object will be put at the end of the array?

Reply:
Is it necessary to follow the order here?

Reply:
Of course the order matters